src/openxr/OpenXRApi.cpp

```diff
@@ -2590,3 +2590,21 @@ Transform OpenXRApi::transform_from_pose(const XrPosef &p_pose, float p_world_sc
 
 	return Transform(basis, origin);
 }
+
+template <typename T>
+Transform _transform_from_space_location(OpenXRApi &api, const T &p_location, float p_world_scale) {
+	const auto flags = XR_SPACE_LOCATION_ORIENTATION_VALID_BIT | XR_SPACE_LOCATION_POSITION_VALID_BIT;
```

Comment:
I think we should break this up. Start with an identify Transform, then:
- if `XR_SPACE_LOCATION_ORIENTATION_VALID_BIT` is set we convert the quarternion to a basis matrix
- if 'XR_SPACE_LOCATION_POSITION_VALID_BIT' is set we assign the origin

That would be the most accurate way of doing this.